docs/layout-of-source-files.rst

```diff
@@ -35,6 +43,35 @@ the exact version of the compiler, so that bugfix releases are still possible.
 It is possible to specify much more complex rules for the compiler version,
 the expression follows those used by `npm <https://docs.npmjs.com/misc/semver>`_.
 
+.. index:: ! pragma, experimental
+
+.. _experimental_pragma:
+
+Experimental Pragma
+-------------------
+
+The second pragma is the experimental pragma. It can be used to enable
+features of the compiler or language that are not yet officially active.
```

Comment:
``not yet officially supported``? ``not yet enabled by default``? ``officially active`` sounds a bit weird, at least to me.